build/common.sh

```diff
@@ -672,14 +672,10 @@ function kube::build::sync_to_container() {
   # they will not be re-generated by 'make'.
   kube::build::rsync \
     --delete \
-    --filter='+ /staging/**' \
-    --filter='- /.git/' \
     --filter='- /.make/' \
     --filter='- /_tmp/' \
     --filter='- /_output/' \
     --filter='- /' \
-    --filter='- zz_generated.*' \
```

Comment:
Because I need a clean repo, and if we don't sync in these generated files, the repo will show them as missing